source/deployment/cluster.rst

```diff
@@ -1,116 +1,115 @@
+High availability in Mattermost consists of running redundant Mattermost application servers, redundant database servers, and redundant load balancers. The failure of any one of these components does not interrupt operation of the system.
 
-This documentation provides a deployment guide, configuration and compatibility details, as well as troubleshooting advice.
+.. contents::
+  :backlinks: top
+  :local:
+  
+Requirements for Continuous Operation
+-------------------------------------
 
+To enable continuous operation at all times, including during server updates and server upgrades, you must make sure that the redundant components are properly sized and that you follow the correct sequence for updating each of the system's components.
 
-- `Deployment Guide <https://docs.mattermost.com/deployment/cluster.html#id1>`_ - **Instructions to set up and maintain high availability on your Mattermost servers.**
- - `Initial Setup Guide for High Availability <https://docs.mattermost.com/deployment/cluster.html#id2>`_
- - `Adding a Server to the Cluster <https://docs.mattermost.com/deployment/cluster.html#id4>`_
- - `Removing a Server from the Cluster <https://docs.mattermost.com/deployment/cluster.html#id9>`_
+Redundancy at anticipated scale
+  Upon failure of one component, the remaining application servers, database servers, and load balancers must be sized and configured to carry the full load of the system. If this requirement is not met, an outage of one component can result in an overload of the remaining components, causing a complete system outage.
 
-- `Configuration and Compatibility <https://docs.mattermost.com/deployment/cluster.html#id13>`_ - **Details on configuring your system for high availability.**
- - `Mattermost Server <https://docs.mattermost.com/deployment/cluster.html#mattermost-server-configuration>`_ - Configure the cluster settings for the Mattermost application server. 
- - `Proxy Server <https://docs.mattermost.com/deployment/cluster.html#proxy-server-configuration>`_ - Configure NGINX or another load balancer to proxy servers in the cluster.
- - `File Storage Location <https://docs.mattermost.com/deployment/cluster.html#file-storage-configuration>`_ - Configure the file storage location to be compatible with a high availability setup.
- - `Database <https://docs.mattermost.com/deployment/cluster.html#database-configuration>`_ - Size and deploy a multi-database configuration for high availability and scaling.
-
-- `Troubleshooting <https://docs.mattermost.com/deployment/cluster.html#id14>`_ - **Advice on troubleshooting your high availability setup.**
-
------
+Continuous single-server update sequence
+  You can apply most configuration changes and dot release security updates without interrupting service, provided that you update the system components in the correct sequence. See the `Upgrade Guide`_ for instructions on how to do this.
 
+In some cases, updates must be applied during a maintenance window
```

Comment:
Can we change this to: 

`Exception: Some updates require maintenance downtime